src/mwoffliner.lib.ts

```diff
@@ -2198,8 +2198,9 @@ async function execute(argv) {
           .replace('</head>',
             genHeaderCSSLink('mobile_main_page_nojs', 'nojs') + '\n' +
             genHeaderCSSLink('style') + '\n' +
-            genHeaderScript('article_list_home') + '\n' +
```

Comment:
@tamasfabi Why have you removed this script?

Reply:
I did not, it's on 2203.

Reply:
👌